elfo/tests/update_config.rs

```diff
@@ -0,0 +1,83 @@
+#![cfg(feature = "test-util")]
+
+use elfo::{config::AnyConfig, messages::ConfigUpdated, prelude::*};
+use elfo_core::messages::{ConfigRejected, UpdateConfig};
```

Comment:
Integration tests in `elfo/tests` shouldn't use `elfo_core`, only `elfo`, because it's a facade.

Reply:
Yep, it's rust-analyzer's shenanigans